main.js

```diff
@@ -98,3 +98,20 @@ function guessResponse2() {
     guessComment2.innerText = "That's Too High!";
   }
 }
+
+var minRange = document.querySelector('.set-min-range');
+var maxRange = document.querySelector('.set-max-range');
+var minRangeText = document.querySelector('.min-range-text-box');
+var maxRangeText = document.querySelector('.max-range-text-box');
+var updateButton = document.querySelector('.range-button');
+
+updateButton.addEventListener('click', updateRange);
+
+minRangeText.addEventListener('input');
```

Comment:
It looks like you have all the necessary JS to complete the story on the Trello card, but one thing I noticed was that you had these two event listeners hear on minRangeText and maxRangeText.  I think with event listeners there is usually some sort of action that comes along with the event that's being listened out for.  So since these don't have that, they aren't necessary here so we can remove these two lines.